docs/team/zognin.md

```diff
@@ -3,34 +3,60 @@ layout: page
 title: Geng Ning's Project Portfolio Page
 ---
 
-### Project: BobTheBodyBuilder
+## Project: BobTheBistroBoss
 
-BobTheBodyBuilder (BTBB) is a **desktop application for private gym managers to manage clients and orders, optimized for use via a command line interface (CLI).**
-It has a GUI (Graphical User Interface) created with JavaFX. It is written in Java, and has about ?? kLoC.
+### Overview
 
+BobTheBistroBoss (BTBB) is a desktop application for home chefs to manage their orders and inventory,
+optimized for use via a command line interface (CLI).
 
-Given below are my contributions to the project.
+### Summary of Contributions
 
-* **New Feature**:
-    * What it does:
-    * Justification:
-    * Highlights:
-    * Credits:
+* **New Feature**: Implemented commands to add and delete ingredients of an order. (Pull request [#92](https://github.com/AY2122S1-CS2103T-W16-2/tp/pull/92)).
+  * What it does: Allows the user to add and delete ingredients of an order.
```

Comment:
Would it look nicer to bold these?

<img width="350" alt="Screenshot 2021-11-02 at 2 32 10 PM" src="https://user-images.githubusercontent.com/77243938/139797561-19984417-df2b-47a2-9f23-03e95b93a9f8.png">

Reply:
Yup